### PR TITLE
Added umaresso.json file

### PR DESCRIPTION
**Pull request by umaresso**
Added umaresso.json file

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1104"></a>

### CODE REVIEW

**Comment by KendallDoesCoding** [comment]
Perfect, issue has been fixed.

**Comment by umaresso** [comment]
Really sorry for the inconvenience , it's almost my first contribution

**Comment by KendallDoesCoding** [comment]
Yeah, no problem. It takes time to learn the strings of Github, and
open-source. We've all made these mistakes when we first started :)

File: public/data/umaresso.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    }
+  ],
+  "milestones": [
+    {
+      "title": "Started YouTube",
+      "image": "https://www.youtube.com/channel/UC61r1YBoNKSiI-c5ezj_FJQ",
+      "date": "Feb 2022",
+      "icon": "youtube",
+      "color": "red",
+      "description": "First video on HTML",
```
Comment (KendallDoesCoding) [inline public/data/umaresso.json:35]:
```suggestion
      "description": "First HTML video",
```

It's fine as it is now, but maybe you can make it this?

**Review by KendallDoesCoding — CHANGES_REQUESTED** [review]

**Review by KendallDoesCoding — COMMENTED** [review]
Do you have a portfolio? If yes, you should definitely link it.

File: public/data/umaresso.json [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+      "icon": "github"
+    },
+    {
+      "name": "Let's Connect on Twitter",
+      "url": "https://twitter.com/umarkhatab465",
+      "icon": "twitter"
+    },
+    {
+      "name": "Learn about Front End Developement",
+      "url": "https://www.youtube.com/channel/UC61r1YBoNKSiI-c5ezj_FJQ",
```
[the remark this answers is not in the document]
Comment (umaresso) [reply public/data/umaresso.json:19]: > Not any changes you have to make with the code, but you need to do it in the youtube studio
> 
> Coz your 100+ subs you can get a custom youtube url, eg: https://www.youtube.com/hashcoder, that will be a short url and much better link to call out here.

Haha ..
Thanks a million buddy !
You are a significant person , making me start the open source journey

**Comment by umaresso** [comment]
> Do you have a portfolio? If yes, you should definitely link it.

I don't have right now, but i'm planning to

File: public/data/umaresso.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (KendallDoesCoding) [reply public/data/umaresso.json:19]: > > Not any changes you have to make with the code, but you need to do it in the youtube studio
> > Coz your 100+ subs you can get a custom youtube url, eg: https://www.youtube.com/hashcoder, that will be a short url and much better link to call out here.
> 
> Haha .. Thanks a million buddy ! You are a significant person , making me start the open source journey

You earned the hundred subs lol. I too just subbed.

File: public/data/umaresso.json [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+      "name": "Follow me on LinkedIn",
+      "url": "https://linkedin.com/in/umarkhatab465",
+      "icon": "linkedin"
+    }
+  ],
+  "milestones": [
+    {
+      "title": "Started YouTube",
+      "image": "https://www.youtube.com/channel/UC61r1YBoNKSiI-c5ezj_FJQ",
+      "date": "Feb 2022",
```
Comment (KendallDoesCoding) [inline public/data/umaresso.json:32]:
```suggestion
      "date": "February 2022",
```

Instead of the short form, its better to put the full month in 😆

**Review by KendallDoesCoding — CHANGES_REQUESTED** [review]

File: public/data/umaresso.json [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+      "url": "https://github.com/umaresso",
+      "icon": "github"
+    },
+    {
+      "name": "Let's Connect on Twitter",
+      "url": "https://twitter.com/umarkhatab465",
+      "icon": "twitter"
+    },
+    {
+      "name": "Learn about Front End Developement",
```
Comment (kabir0x17) [inline public/data/umaresso.json:18]: Incorrect spelling 
```suggestion
      "name": "Learn about Front End Development",
```

**Review by kabir0x17 — CHANGES_REQUESTED** [review]

**Pull request merged**